ote_sdk/ote_sdk/entities/dataset_item.py

```diff
@@ -247,13 +247,15 @@ def get_annotations(
         self,
         labels: Optional[List[LabelEntity]] = None,
         include_empty: bool = False,
+        include_ignored: bool = False,
     ) -> List[Annotation]:
         """
         Returns a list of annotations that exist in the dataset item (wrt. ROI). This is done by checking that the
         center of the annotation is located in the ROI.
 
         :param labels: Subset of input labels to filter with; if ``None``, all the shapes within the ROI are returned
         :param include_empty: if True, returns both empty and non-empty labels
+        :param include_ignored: if True, includes the labels in self.__ignored_labels
```

Comment:
`__ignored_labels` is a protected method, an implementation detail that should not appear in the docstring of a public method. You can just say "include the ignored labels"